new_issue_template.md

```diff
@@ -0,0 +1,33 @@
+## Issue Description
+**Issue Type:**
+- [ ] Bug
+- [ ] Feature Request
+
+**Summary:**
+[Provide a brief description of the issue.]
+
+**Expected Behavior:**
+[Explain what you expected to happen.]
+
+**Actual Behavior:**
+[Explain what actually happened.]
+
+## Steps to Reproduce
```

Comment:
We need to update the content of the template.

As if it will be a feature, then there won't be any "steps to reproduce"

Can you please check how other open source projects are doing it?

Reply:
okay..I'll soon update this ,any other review??

Reply:
Will that be fine if I add two different sections under details depending on whether they select "Bug" or "Feature Request," they can expand the relevant details section, which is enclosed in a <details> element?